app/src/main/assets/config.json

```diff
@@ -32,6 +32,8 @@
   "mSpo2": true,
   "mBMI": true,
   "mResp": true,
+  "privacy_boolean": true,
+  "privacy_value": "This is demo Raunaq VSO privacy notice text values",
```

Comment:
Use some sensible demo text. You can use the one that was provided to you. Also this should be a string so that it can be translated into multiple languages